oxide-auth/src/endpoint/tests/pkce.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "client-map")]
```

Comment:
Disabling the test suite is not a good implementation. We don't want to loose assurance due to a smaller feature selection. Can the dev-dependencies be extended accordingly to support this?

Reply:
Generalized the `ClientMap` impl and this test now uses a `NoopPasswordPolicy` which just concatenates the Client ID and passphrase and checks for equality.  

Good enough for tests.